lib/js_of_ocaml/json.ml

```diff
@@ -51,13 +138,18 @@ let input_reviver =
   in
   wrap_meth_callback reviver
 
-let unsafe_input s = json##parse_ s input_reviver
+let unsafe_input s =
+  match Sys.backend_type with
+  | Other "wasm_of_ocaml" -> failwith "Json.unsafe_input: not implemented"
```

Comment:
Can you add a comment explaining why it's not implemented ?

Reply:
I changed to “not available in the Wasm backend”. What do you think?

Reply:
What about a comment answering these questions ?
Any reason it is not implemented ? Is an implementation possible ? Is it planned?

Reply:
I would assume it could be implemented, but is not necessary to run existing packages and has not been a priority. @vouillon would know better.

Reply:
Function `output` is used by Eliom and in some Jane Street code, but `unsafe_input` does not seem to be used anywhere.
Also, we do not know whether the JSON value should be deserialized as a JS value or an OCaml value.

Reply:
And the encoding of OCaml values is ambiguous since both integers and floats are mapped to numbers.

Reply:
I have updated the error message to try to summarize that.

Reply:
I got confused between. “comment” and “error message”, sorry. I see you fixed that.